src/sentry/interfaces/base.py

```diff
@@ -89,6 +89,8 @@ def __setstate__(self, state):
             self._data = {}
 
     def __getattr__(self, name):
+        if name == '__bases__':
+            return super(Interface, self).__getattr__(name)
```

Comment:
What's this about?

Reply:
had to do this to get `isinstance` to work :-/

Reply:
actuallyyyy maybe i'm wrong and was messing something else up

Reply:
yup